blocks/article-tag-block/features/tag/default.test.jsx

```diff
@@ -53,26 +48,26 @@ describe("the article tag block", () => {
 				lazyLoad: true,
 			};
 			const wrapper = mount(<ArticleTags customFields={config} />);
-			expect(wrapper.html()).toBe(null);
+			expect(wrapper.isEmptyRender()).toBe(true);
 		});
 
 		it("should render a parent container for the tags", () => {
 			const { default: ArticleTags } = require("./default");
```

Comment:
Are we able to convert these tests to use the `import` syntax instead of the `require` syntax?

Reply:
yep updated!

Reply:
yep updated! thanks for the feedback @matthewroach